jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/params/StoreParamsBuilder.java

```diff
@@ -229,10 +241,39 @@ public int getNodeMissCacheSize() {
     }
 
    public StoreParamsBuilder nodeMissCacheSize(int nodeMissCacheSize) {
-       NodeMissCacheSize = new Item<>(nodeMissCacheSize, true);
+       this.NodeMissCacheSize = new Item<>(nodeMissCacheSize, true);
        return this;
    }
 
+   public int getPrefixNode2NodeIdCacheSize() {
+       return prefixNode2NodeIdCacheSize.value;
+   }
+
+  public StoreParamsBuilder prefixnode2NodeIdCacheSize(int prefixNode2NodeIdCacheSize) {
+      this.prefixNode2NodeIdCacheSize = new Item<>(prefixNode2NodeIdCacheSize, true);
+      return this;
+  }
+
+   public int getPrefixNodeId2NodeCacheSize() {
+       return prefixNodeId2NodeCacheSize.value;
+   }
+
+  public StoreParamsBuilder prefixnodeId2NodeCacheSize(int prefixNodeId2NodeCacheSize) {
+      this.prefixNodeId2NodeCacheSize = new Item<>(prefixNodeId2NodeCacheSize, true);
+      return this;
+  }
+
+   public int getPrefixNodeMissCacheSize() {
+       return prefixNodeMissCacheSize.value;
+   }
+
+  public StoreParamsBuilder prefixNodeMissCacheSize(int prefixNodeMissCacheSize) {
+      this.prefixNodeMissCacheSize = new Item<>(prefixNodeMissCacheSize, true);
+      return this;
+  }
+
+
+
```

Comment:
Extra blank lines :point_up:

Reply:
Done (there is other strange white space as well - no idea how it came about (indent of 2 ??!!!))